src/py42/services/_connection.py

```diff
@@ -247,7 +248,13 @@ def _handle_error(method, url, response):
     try:
         response.raise_for_status()
     except HTTPError as ex:
-        raise_py42_error(ex)
+        if response.json()["error"][0]["primaryErrorKey"] in [
```

Comment:
we currently need to support versions of `requests` where the `.json()` method doesn't exist :-(.
The easiest way to port this is:
```python
response_dict = json.loads(response.text)
```

Reply:
Updated in 3513cf0